samples/nrf9160/modem_shell/prj.conf

```diff
@@ -45,6 +45,7 @@ CONFIG_FPU=y
 CONFIG_CJSON_LIB=y
 
 # Getopt
+CONFIG_DEVMEM_SHELL=n
```

Comment:
It seems that this is needed to make `getopt` work. However, it's not obvious why this option is related to `getopt`. It would be good to have a comment here.

Reply:
I updated my PR with a comment. The problem is that this `devmem shell` library **selects** Zephyr's version of `getopt` in KConfig.